RETL - Features overview and Models guide (#1034)
rudderlabs/rudderstack-docs: docs/sources/reverse-etl/common-settings/models.mdx

</ul>
</div>

## Creating a model

Follow these steps to create a model:

> 1. Login to the <a href="https://app.rudderstack.com/">RudderStack dashboard</a>.

Reviewer: ```suggestion
1. Log in to the <a href="https://app.rudderstack.com/">RudderStack dashboard</a>.
```

> 2. Click **Models** in the left navigation bar and select **New model**.

Reviewer: I think we have a new flow - now it's **Enhance** > **Models**.

> 2. Click **Models** in the left navigation bar and select **New model**.

Reviewer: ```suggestion
2. From the left navigation bar, go to **Enhance** > **Models** and click **New model**.
```
